### PR TITLE
Add type declarations for npm's "timespan" module

### PR DESCRIPTION
**Pull request by knocte**

### CODE REVIEW

File: types/timespan/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// Type definitions for timespan 2.3
+// Project: https://github.com/indexzero/TimeSpan.js
+// Definitions by: Andres G. Aragoneses <https://github.com/knocte>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "timespan" {
```
Comment (DanielRosenwasser) [inline types/timespan/index.d.ts]: Remove this and the respective `}`

File: types/timespan/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+export class TimeSpan {
+    constructor(milliseconds: any, seconds: any, minutes: any, hours: any, days: any);
```
Comment (DanielRosenwasser) [inline types/timespan/index.d.ts]: Are these all really `any`? They're clearly all intended to be `number`

File: types/timespan/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
```
Comment (DanielRosenwasser) [inline types/timespan/tsconfig.json]: Set `esModuleInterop` to `true`

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
This needs tests.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**